src/Sulu/Component/Rest/ListBuilder/AbstractListBuilder.php

```diff
@@ -238,8 +238,19 @@ public function search($search)
      */
     public function sort(AbstractFieldDescriptor $fieldDescriptor, $order = self::SORTORDER_ASC)
     {
-        $this->sortField = $fieldDescriptor;
-        $this->sortOrder = $order;
+        $this->sortFields = array($fieldDescriptor);
```

Comment:
Why should `sort` overrride the previous value? `where` is also not working like that...